README.md

```diff
@@ -409,6 +412,42 @@ api.campaigns.actions.get({
 });
 ```
 
+## Create Campaign Action
+
+> http://docs.oracle.com/cloud/latest/marketingcs_gs/OMCGF/op-sites-%7Bsite-id%7D-sandbox-campaigns-%7Bcampaign-id%7D-actions-post.html
+
+```javascript
+api.campaigns.actions.create({
+  siteName: 'test.com',
+  campaignName: 'a31-test',
+  elementName: 'renderer',
+  // siteId: 'MzIxMzM',
+  // campaignId: 'MDA2MjYx',
+  // elementId: 'MDMyMDU4',
```

Comment:
`elementId` and `elementName` aren't actually part of the actions request schema.